tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +122,162 @@ module('Acceptance | guid file | registration files', hooks => {
+                .containsText(funder.award_title, `Funder award title is unchanged for ${funder.funder_name}.`);
+            // Award URI
+            assert.dom('[data-test-target-funder-award-uri-label]').hasText('Award URI',
+                'Funder award URI label is properly set.');
+            assert.dom(`[data-test-target-funder-award-uri="${funder.funder_name}"]`)
+                .containsText(funder.award_uri, `Funder award URI is unchanged for ${funder.funder_name}.`);
+            // Award number
+            assert.dom('[data-test-target-funder-award-number-label]').hasText('Award number',
+                'Funder award number label is properly set.');
+            assert.dom(`[data-test-target-funder-award-number="${funder.funder_name}"]`)
+                .hasText(funder.award_number, `Funder award number is unchanged for ${funder.funder_name}`);
+        }
+        // Target title
+        assert.dom('[data-test-target-title-label]').hasText('Title',
+            'File metadata target title text is properly set.');
+        assert.dom('[data-test-target-title]').exists();
+        // Target description
+        assert.dom('[data-test-target-description-label]').hasText('Description',
+            'File metadata target description text is properly set.');
+        assert.dom('[data-test-target-description]').exists();
+    });
+
+    // Node type
+    test('Node type', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        assert.dom('[data-test-metadata-node]').hasText('Registration Metadata',
+            'Node noun for registration properly set.');
+    });
+
+    // Save and cancel buttons with write permissions
+    test('Save and cancel', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify download
+        assert.dom('[data-test-download-button]').exists();
+
+        // Verify edit form
+        await click('[data-test-edit-metadata-button]');
+        assert.dom('[data-test-title-field]').exists();
+        assert.dom('[data-test-description-field]').exists();
+        assert.dom('[data-test-select-resource-type]').exists();
+        assert.dom('[data-test-select-resource-language]').exists();
+        assert.dom('[data-test-cancel-editing-metadata-button]').exists();
+        assert.dom('[data-test-save-metadata-button]').exists();
+        await click('[data-test-cancel-editing-metadata-button]');
+        assert.dom('[data-test-edit-metadata-form]').doesNotExist();
+        await click('[data-test-edit-metadata-button]');
+
+        // Screenshot before changes
+        await percySnapshot(assert);
+        // Update title
+        const editTitleTextArea = document.querySelectorAll('textarea')[0];
+        editTitleTextArea.textContent = 'A test title.';
```

Comment:
You shouldn't need to do this. `await fillIn('[data-test-title-field]', 'A test title.');` or similar should work here.

Reply:
I actually was just porting this over from the previous ticket here: https://github.com/CenterForOpenScience/ember-osf-web/pull/1793 but it was giving me an error message. I can undo changes to replicate it. One moment.

Reply:
<img width="1196" alt="image" src="https://user-images.githubusercontent.com/82047646/219126646-d8ecd0c7-3d97-4aac-9eeb-bae865ee62bc.png">

Reply:
I suspect that it throws an error because you are trying to apply `fillIn()` on the element selected by selector `[data-test-title-field]`, which is a `<div>` instead of an input field. If you look at the actual rendered DOM, the `textarea` input field is nested within the div. So you will need to change your selector.

Also, changing the `textContent` property of a `textarea` element is not actually inputting anything to the input field. Instead, it is changing the text content between the element tags of the `textarea` element. (See [here](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent)) If you want to update the actual value inputted into the input field, you should update the `value` property of the `textarea` element, which is done by using the `fillIn()` method.

Reply:
The helper function fillIn is now working and the update title and description tests have been also been updated.

Reply:
@adlius I had mentioned that. The tldr is we did not know why fillIn was not correctly working. I had asked if fillIn used value, textContent or innerText because I had tried value's alternatives but value was working. If fillIn relied on value, it should have be working. I'm glad to see it is.